target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -191,6 +191,34 @@ define Device/tplink_archer-d50-v1
 endef
 TARGET_DEVICES += tplink_archer-d50-v1
 
+define Device/tplink_archer-d7-v1
+  $(Device/tplink-qca-dsl)
+  ATH_SOC := qca9558
```

Comment:
Please rename this to "SOC" due to 95030ac0b673165772daf27128359eaaf536d457.

Reply:
Fixed!